ctc_decoder/beam_search.py

```diff
@@ -75,8 +81,8 @@ def beam_search(mat: np.ndarray, labels: str, beam_width: int = 25, lm: Optional
     last = BeamState()
     labeling = ()
     last.entries[labeling] = BeamEntry()
-    last.entries[labeling].pr_blank = 1
-    last.entries[labeling].pr_total = 1
+    last.entries[labeling].pr_blank = LOG_ZERO
```

Comment:
should be log(1)==0 instead of LOG_ZERO which is -inf

Reply:
That's true, that allows the below if statements to be cleaned up also, thanks for picking this up.